packages/org-unit-dialog/src/OrgUnitSelector.js

```diff
@@ -43,6 +44,7 @@ class OrgUnitSelector extends Component {
                     counter += 1;
 
                     if (counter > 1) {
+                        // eslint-disable-next-line
```

Comment:
Is there a danger of infinite updating with setState in CDU?

Reply:
Yes, unless "if" condition is implemented.